client/src/types/User.ts

```diff
@@ -0,0 +1,4 @@
+export type User = {
+  id: string;
+  token: string;
+};
```

Comment:
What is a `User` in the context of the app? 😄 

I'd maybe do something like this instead:


```suggestion
export type RsvpAuth = Pick<Rsvp, "id" | "token">;
```

Although I'm not 100% on the name of this type. It should represent basically the minimal details that the client stores to be able to edit an existing RSVP 🤔